coriolis/osmorphing/redhat.py

```diff
@@ -156,15 +157,23 @@ def set_net_config(self, nics_info, dhcp):
         self._add_net_udev_rules(net_ifaces_info)
 
     def _yum_install(self, package_names, enable_repos=[]):
-        yum_cmd = 'yum install %s -y%s' % (
-            " ".join(package_names),
-            "".join([" --enablerepo=%s" % r for r in enable_repos]))
-        self._exec_cmd_chroot(yum_cmd)
+        try:
+            yum_cmd = 'yum install %s -y%s' % (
+                " ".join(package_names),
+                "".join([" --enablerepo=%s" % r for r in enable_repos]))
+            self._exec_cmd_chroot(yum_cmd)
+        except exception.CoriolisException as err:
+            raise exception.FailedPackageInstallationException(
```

Comment:
Please add the same error message wrapper mentioned above for apt-based systems.